cli/auth.go

```diff
@@ -129,15 +129,15 @@ func (c *viamClient) loginAction(cCtx *cli.Context) error {
 		already := "Already l"
 		if !alreadyLoggedIn {
 			already = "L"
-			viamLogo(cCtx.App.Writer)
+			viamLogo(cCtx.App.ErrWriter)
 		}
 
 		printf(cCtx.App.Writer, "%sogged in as %q, expires %s", already, t.User.Email,
 			t.ExpiresAt.Format("Mon Jan 2 15:04:05 MST 2006"))
 	}
 
 	if _, isAPIKey := c.conf.Auth.(*apiKey); isAPIKey {
-		warningf(c.c.App.Writer, "was logged in with an api-key. logging out")
+		warningf(c.c.App.ErrWriter, "was logged in with an api-key. logging out")
```

Comment:
Justification: this seems like diagnostic info and is not really part of the program output. 

I am ~50% confident about this one.

Reply:
Same.